php/helpers.php

```diff
@@ -44,46 +44,48 @@ function block_field( $name, $echo = true ) {
 	}
 
 	// Cast block value as correct type.
-	switch ( $block_lab_config->fields[ $name ]->type ) {
-		case 'string':
-			$value = strval( $value );
-			break;
-		case 'textarea':
-			$value = strval( $value );
-			if ( isset( $block_lab_config->fields[ $name ]->settings['new_lines'] ) ) {
-				if ( 'autop' === $block_lab_config->fields[ $name ]->settings['new_lines'] ) {
-					$value = wpautop( $value );
+	if ( isset( $block_lab_config->fields[ $name ]->type ) ) {
+		switch ( $block_lab_config->fields[ $name ]->type ) {
+			case 'string':
```

Comment:
The diff inside this `switch` block is whitespace-only, from indenting it inside the new conditional